src/initiate.rs

```diff
@@ -36,7 +36,7 @@ fn deflag_new_atoms(mut commands: Commands, query: Query<Entity, With<NewlyCreat
 pub struct InitiatePlugin;
 impl Plugin for InitiatePlugin {
     fn build(&self, app: &mut App) {
-        app.add_system(deflag_new_atoms.in_base_set(CoreSet::Update));
+        app.add_systems(Update, deflag_new_atoms);
```

Comment:
This relies on an implicit likeness between the SystemSet used for inserting atoms into the world (in atom_sources/mod.rs), so that structural changes are delayed and the NewlyCreated flag survives a full frame. I think that is still the case with bevy, but it might be good to move the deflagging either to PreUpdate or define a stricter relation (e.g, define the SystemStage to use in `initiate.rs` and reference it from `atom_sources/mod.rs`